scripts/utils.py

```diff
@@ -188,8 +189,12 @@ def load_data(path):
   return yaml_load(os.path.join(data_dir(), path))
 
 def save_data(data, path):
-  return yaml_dump(data, os.path.join(data_dir(), path))
-
+  yaml_dump(data, os.path.join(data_dir(), path))
+  
+  if (path=="legislators-historical.yaml"):
+    generate_alternate_formats(historical=True)
+  else:
+    generate_alternate_formats(historical=False)
```

Comment:
Will this cause *all* data files to be regenerated if even one data file changes?

What I'd originally suggested is that when a specific data file is saved, that we save the CSV/JSON version of that data file right after we save the YAML version. I think the code would be simpler, and we could avoid a `historical` boolean flag, by taking that approach.